JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreens.kt

```diff
@@ -170,17 +160,29 @@ fun HomeListDetailScreen(
 }
 
 /**
- * The home screen displaying a list.
+ * A [Modifier] that tracks all input, and calls [block] every time input is received.
+ */
+private fun Modifier.notifyInput(block: () -> Unit): Modifier =
+    pointerInput(block) {
```

Comment:
You might not want to reset this `pointerInput` if the lambda changes. What about passing `Unit` as `key` and the protecting the block for updates?

```suggestion
    val blockState by rememberUpdatedState(block)
    pointerInput(Unit) {
```

Reply:
Updated! Thanks for the explanation offline as well for why we want to do this!